SwissTransferFeatures/NewTransferView/NewTransferSettingsView.swift

```diff
@@ -33,17 +36,23 @@ struct NewTransferSettingsView: View {
     @State private var isShowingDownloadLimitSetting = false
     @State private var isShowingLanguageSetting = false
 
+    public init() {
+        @InjectService var settingsManager: AppSettingsManager
+        _appSettings = StateObject(wrappedValue: FlowObserver(flow: settingsManager.appSettings))
+    }
+
     var body: some View {
         VStack(alignment: .leading, spacing: IKPadding.medium) {
             Text(STResourcesStrings.Localizable.advancedSettingsTitle)
                 .font(.ST.callout)
                 .foregroundStyle(Color.ST.textPrimary)
 
             VStack(alignment: .leading, spacing: IKPadding.medium) {
+                let validityItem = settingItem(setting: .validityPeriod)
```

Comment:
I think you can _later_ on your PR use a `foreach` here, but that would require further refactor work that is beyond my settings screen IMO.

Reply:
Discussed with @valentinperignon actually no need to go for a "foreach"